lms/templates/courseware/course_navigation.html

```diff
@@ -83,27 +78,14 @@
 % if disable_tabs is UNDEFINED or not disable_tabs:
 <nav class="${active_page} wrapper-course-material" aria-label="${_('Course Material')}">
   <div class="course-material">
-    <ol class="course-tabs">
-      % for tab in get_course_tab_list(request, course):
-        <%
-            tab_is_active = (tab.tab_id == active_page) or (tab.tab_id == default_tab)
-        %>
-          <li>
-             <a href="${tab.link_func(course, reverse) | h}" class="${url_class(tab_is_active)}">
-                 ${_(tab.name) | h}
-                 % if tab_is_active:
-                   <span class="sr">, current location</span>
-                 %endif
-                 % if tab_image:
-                   ## Translators: 'needs attention' is an alternative string for the
-                   ## notification image that indicates the tab "needs attention".
-                   <img src="${tab_image}" alt="${_('needs attention')}" />
-                 %endif
-             </a>
-          </li>
-      % endfor
-      <%block name="extratabs" />
-    </ol>
+     <%
+      base_tabs_tpl = 'courseware/tabs.html'
+      tabs_tmpl = static.get_template_path(base_tabs_tpl)
+      if tabs_tmpl == base_tabs_tpl:
+          tabs_tmpl = '/' + base_tabs_tpl
```

Comment:
@ziafazal I've tried but it doesn't work. The main problem is that **<%include** works only with the slash in the beginning of the line (example: https://github.com/edx/edx-platform/blob/90f2fb868e0646733fad8ec615af2d2dec11801e/lms/templates/static_htmlbook.html#L122 )
But including templates from microsites works through the *optional_include_mako* that works only with relative pathes (example: https://github.com/edx/edx-platform/blob/90f2fb868e0646733fad8ec615af2d2dec11801e/lms/templates/main.html#L125 )
otherwise TemplateLookupException will be raised